src/cc/bcc_elf.c

```diff
@@ -0,0 +1,269 @@
+#include <sys/types.h>
```

Comment:
I like the direction a lot. Please add apache2 banner to all new .h and .c
Also we need to agree on formatting of .c files. Take a look at existing style.
I don't think it's pretty (I prefer kernel style with tabs), but that's what bcc is using already. So better be consistent. @drzaeus77 we need pre-commit lint checker.